Library/Homebrew/extend/os/mac/tap.rb

```diff
@@ -0,0 +1,11 @@
+# typed: true
+# frozen_string_literal: true
+
+class Tap
+  def self.install_default_cask_tap_if_necessary
```

Comment:
A `false` definition for non-macos would be good here.

Reply:
`false` is the default implementation.

Reply:
Cool, thanks!